addons/csw/stringtable.xml

```diff
@@ -652,6 +708,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_metis_tube">
             <English>[CSW] 9k115 Metis Launcher Bag</English>
+            <German>[CSW] 9k115 Metis  Werfer Tasche</German>
```

Comment:
Das K (bzw M weiter unten) is groß. Leerzeichen is doppelt. Ansonsten immer noch das gleiche Problem wie mit den anderen Taschen